orchagent/saihelper.cpp

```diff
@@ -82,6 +82,7 @@ sai_dash_inbound_routing_api_t*     sai_dash_inbound_routing_api;
 sai_dash_eni_api_t*                 sai_dash_eni_api;
 sai_dash_vip_api_t*                 sai_dash_vip_api;
 sai_dash_direction_lookup_api_t*    sai_dash_direction_lookup_api;
+sai_twamp_api_t*            sai_twamp_api;
```

Comment:
Please align this with the previous code.

Reply:
Done